curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-describe-places-and-events/66a95c8ff92dcc4ed83a8df9.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 66a95c8ff92dcc4ed83a8df9
+title: Task 26
+challengeType: 19
+dashedName: task-26
+---
+<!--
+AUDIO REFERENCE:
+Bob: How many people usually participate in these events?
+-->
+
+# --description--
+
+`How many` is used to ask about the number of something.
+
+`Usually` indicates something that happens regularly. 
+
+Questions in the Present Simple tense are used to ask about habits, facts, or general truths. For example:
+
+- `How many books do you read in a month?` (This question asks about the regular number of books someone reads.)
+- `How many people usually attend the meeting?` (This question asks about the typical number of people at a regular meeting.)
+
+# --question--
+
+## --text--
+
+What does Bob want to know?
+
+## --answers--
+
+How often the events are held
+
+### --feedback--
+
+Bob is not asking about the frequency of the events.
+
+---
+
+How many people typically participate in the events
+
+---
+
+How long the events tipically last
```

Comment:
```suggestion
How long the events typically last
```

Reply:
Hello, Tom! Thank you for your review. I'll reply to your comments here:

- Periods at the end: Talked to Daniel and here's what he suggested:
 Is the alternative a sentence/phrase? **It takes normal punctuation.**
Is the alternative just, for example, a noun with an adjective? **No punctuation**. 

- Multi-questions challenges: That would be amazing! Looking forwards to this change. 

- True or False consistency: Agreed. I think I'll adopt "Is it true or false:" It's short, simple and easy to understand. 

- Mixed instructions:  In this curriculum I'm trying new approach so we can test more complex thinking skills. Many type 19 will have an audio and ask comprehension questions instead of asking to select what the caracter is saying. The example you mentioned is a good one to illustrate. 
 In the task "Listen to the audio and answer: What does Bob want to know from Lisa?" the options are not about what Bob is saying, they are about his intention with the question. In this kind of question we can't just say "Listen to the audio and complete the sentence" or other kind of standard phrase. We have to be specific of what we want the student to pay attention to answer. 

- I agree with your point about consistency. I'll review the block so these instructions are within the # --text -- section not the # --instructions-- 
- I also agree with the idea of a document for standards to follow. This first block and everyone's feedback will be ideal to do that. Thank you so much!

Next steps: I'll now review the individual comments you and Daniel made and add a new commit with the changes.

Reply:
Hello, Tom! Thank you for your review. I'll reply each topic you mentioned below:

- Periods at the end: Talked to Daniel and here's what he suggested:
Is the alternative a sentence/phrase? *It takes normal punctuation.*
Is the alternative just, for example, a noun with an adjective? *No punctuation.*

Multi-questions challenges: That would be amazing! Looking forwards to this change.

True or False consistency: Agreed. I think I'll adopt "Is it true or false:" It's short, simple and easy to understand.

Mixed instructions: In this curriculum I'm trying new approach so we can test more complex thinking skills. Many type 19 will have an audio and ask comprehension questions instead of asking to select what the caracter is saying. I like the idea of adding #--instructions-- to the type 19 and keep #--text-- only with the question.
I'll review the block so these instructions are within the # --instructions -- section not the # --text--

- I also agree with the idea of a document for standards to follow. This first block and everyone's feedback will be ideal to do that. Thank you so much!

Next steps: I'll now review the individual comments you and Daniel made and add a new commit with the changes.